test/csv/test_csv_filter.rb

```diff
@@ -78,6 +78,17 @@ def test_option_input_col_sep
                  run_csv_filter(csv, "--input-col-sep=:"))
   end
 
+  def test_option_input_quote_char
+    input_quote_char = "'"
+    str = CSV.generate(quote_char: input_quote_char) do |csv|
+      csv << ['foo', 0]
+      csv << ["'bar'", 1]
+      csv << ['"baz"', 2]
+    end
+    assert_equal(["foo,0\n" + "'bar',1\n" + "\"\"\"baz\"\"\",2\n", ""],
+                 run_csv_filter(str, "--input-quote-char=#{input_quote_char}"))
```

Comment:
How about using here-document for easy to read?

```suggestion
    assert_equal([<<-OUTPUT, ""], run_csv_filter(str, "--input-quote-char=#{input_quote_char}"))
foo,0
'bar',1
"""baz""",2
    OUTPUT
```